### DISCUSSION TITLE
Form fields inside a table

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6503] [Help] [answered]
Is it possible to have form fields inside a table, to be able to update data straight into the table ?

I tried to create a number field, and put the default value to {{ tbl_ruptures.RUPTURES.Id_rupture }} but as you can see it remains on the value of the first row...

![image](https://user-images.githubusercontent.com/28592069/176485448-a86fdee1-3b2c-4c76-b5f4-9693a2f2622a.png)

**Comment by melohagan** [comment]
HI @NicolasFabre81 

What happens if you try `{{ tbl_ruptures.RUPTURES.0.Id_rupture }}` assuming RUPTURES is a relationship field type.

**Reply by NicolasFabre81** [reply]
Hello @melohagan , I'm facing this problem again (my last message on discord)
RUPTURES is a postgreSQL table.
I'd just like to display my table using a FORM/DP/RPT structure, and put updatable fields to update the table.
Isn't it possible ?

**Comment by NicolasFabre81** [comment, marked as answer]
I have to answer myself : we just need to have the form INSIDE the repeater !

**Reply by melohagan** [reply]
Brilliant - good job Nicolas!
